exp3/TTS_dataset/VAD.py

```
@@ -0,0 +1,243 @@
+# 最小语音片段长度（毫秒）。小于此长度的语音片段将被忽略。
+MIN_SPEECH_SEGMENT_MS = 50 
+
+
+def convert_audio_to_wav_format(input_path, temp_path):
+    """使用pydub将音频文件转换为VAD兼容的格式"""
+    audio = AudioSegment.from_file(input_path)
+    
+    # 转换为单声道
+    if audio.channels > 1:
+        audio = audio.set_channels(1)
+    # 转换为16kHz采样率（WebRTC VAD推荐）
```

Ruff (0.14.5)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)